packages/apps/registry-backend/.env.example

```diff
@@ -1,6 +1,8 @@
 PORT=3000
-MONGODB_URI=mongodb://localhost:27017/registry
+MONGODB_URI=mongodb://localhost:27017/registry?replicaSet=rs0
```

Comment:
mongo wouldn't let me save anything until i turned on replica sets by adding this to `/opt/homebrew/etc/mongod.conf`

```
replication:
  oplogSizeMB: 2000
  replSetName: rs0
```

and then running `rs.initiate()` in the mongosh REPL